Avoid useless reload of the page

laulaz opened the pull request:
We had a UX issue with page reloading after cookie acceptation.
After reading the code, we don't understand why the page had to be reloaded to activate cookies.
It's working fine without that reload.

Can you please merge this PR or explain the edge case you were wanting to handle with that reload ?

laulaz committed:
Fill missing history (from previous PR)

laulaz committed:
Avoid useless reload of the page

Cookies are working without it

laulaz commented:
@kingel @ralphjacobs could you please take a look at this PR ?

kingel commented:
@laulaz We don't do a lot of active plone development anymore, if you want I could give you access to the repo and to pypi

laulaz commented:
@kingel Thank you for your quick answer !
I'm OK to have Pypi access (user : laz).
For the repo, would you mind to transfer it to collective organization to get more maintainers / developers working on it ?  Otherwise, I'm OK with access to repo ;-)

Thanks a lot !

kingel commented:
@laulaz github promised to take the pr's and issues as well but no go.. I moved it to https://github.com/collective/collective.cookiecuttr so maybe you can redo your pr

I added you on pypi

laulaz commented:
@kingel 

> I added you on pypi

Thanks !

> @laulaz github promised to take the pr's and issues as well but no go.. I moved it to https://github.com/collective/collective.cookiecuttr so maybe you can redo your pr

Did you Transfer it ? it's strange that the repo is duplicated now

kingel commented:
yes i transfered it but as you say things are weird.. i dont have time to look into that. I will archive this repo
